doc/api/zlib.markdown

```diff
@@ -109,6 +109,30 @@ http.createServer((request, response) => {
 }).listen(1337);
 ```
 
+By default, the zlib methods will throw an error when decompressing truncated
+data. If you know that your input may be incomplete, or you want to
+inspect only the beginning of a compressed file, you can change the flushing
+method which is used to compressed the last chunk of input data:
+
+```js
+// This is a truncated version of the buffer from the above examples
+const buffer = new Buffer('eJzT0yMA', 'base64');
+
+zlib.unzip(buffer, { finishFlush: zlib.Z_SYNC_FLUSH }, (err, buffer) => {
+  if (!err) {
+    console.log(buffer.toString());
+  } else {
+    // handle error
+  }
+});
+```
+
+This will not change the behaviour in other error-throwing situations, e.g.
+when the input data has an invalid format. You will also not be able to tell
+whether the input ended prematurely or not, including possibly missing
+integrity checks, so you will have to make sure that the decompressed result
+is valid yourself.
```

Comment:
Again, I'd prefer to avoid using `you` ... perhaps:

```
This will not change the behavior in other error-throwing situations, e.g.
when the input data has an invalid format. Using this method, it will not be
possible to determine whether the input ended prematurely. It is also possible
to miss integrity checks making it necessary to manually check that the 
decompressed result is valid.
```